src/ofxstatement/plugins/iso20022.py

```diff
@@ -81,13 +87,17 @@ def _get_statement_el(self, tree: ET.ElementTree) -> ET.Element:
     def _parse_statement_properties(self, tree: ET.ElementTree) -> None:
         stmt = self._get_statement_el(tree)
 
-        bnk = stmt.find("./s:Acct/s:Svcr/s:FinInstnId/s:BIC", self.xmlns)
+        bnk = stmt.find("./s:Acct/s:Svcr/s:FinInstnId/s:BICFI", self.xmlns)  # checked
         if bnk is None:
-            bnk = stmt.find("./s:Acct/s:Svcr/s:FinInstnId/s:Nm", self.xmlns)
-        iban = stmt.find("./s:Acct/s:Id/s:IBAN", self.xmlns)
-        assert iban is not None
-        ccy = stmt.find("./s:Acct/s:Ccy", self.xmlns)
-        bals = stmt.findall("./s:Bal", self.xmlns)
+            bnk = stmt.find("./s:Acct/s:Svcr/s:FinInstnId/s:Nm", self.xmlns)  # checked
+
+        ibanfind = stmt.find(
+            "./s:Ntry/s:NtryDtls/s:TxDtls/s:RltdPties/s:CdtrAcct/s:Id/s:IBAN",
+            self.xmlns,
+        )  # checked, appears to not be included in every camt.053 statement?
+        # assert iban is not None
+        ccy = stmt.find("./s:Acct/s:Ccy", self.xmlns)  # checked
+        bals = stmt.findall("./s:Bal", self.xmlns)  # checked
```

Comment:
what `#checked` comment means?

Reply:
I have no idea and \#python@libera doesn’t seem to know either (so far). It’s from @RealStickman’s code.
@RealStickman, can you explain?

Reply:
It's been a while since I wrote that, but I think the `# checked` was when I went through the XML manually to compare with the converter. Those I confirmed to be present got that comment.

Reply:
I removed them because IMHO they don’t provide much value as everything in that code should be expected to be checked in some way. Guesses, assumptions, and other thing "not checked" should be commented instead.